src/macaron/slsa_analyzer/package_registry/npm_registry.py

```diff
@@ -128,20 +133,28 @@ def download_attestation_payload(self, url: str, download_path: str) -> bool:
 
         * publish with "https://github.com/npm/attestation/tree/main/specs/publish/v0.1" predicateType
         * SLSA with "https://slsa.dev/provenance/v0.2" predicateType
+        * SLSA with "https://slsa.dev/provenance/v1" predicateType
```

Comment:
If the SLSA provenance v1 is added, I think we should change the above line to `Each npm package has three types of attestations`.

Reply:
See commit [bae2fe0](https://github.com/oracle/macaron/pull/551/commits/bae2fe0376631a0a9081d8d507459bf5f14b6d5f).